src/include/distributed/pg_dist_partition.h

```diff
@@ -23,6 +23,7 @@ typedef struct FormData_pg_dist_partition
 {
 	Oid logicalrelid;              /* logical relation id; references pg_class oid */
 	char partmethod;               /* partition method; see codes below */
+	int colocationid;
```

Comment:
@anarazel in this struct order of fields is different than order of columns in pg_dist_partition table. I put this field before partkey because partkey is variable-length field. Is there any problem because of the order difference? If there is, I will have to recreate pg_dist_partition table and copy data in upgrade script.